perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AuthzResolverBlImpl.java

```diff
@@ -1922,4 +1964,172 @@ private static void prepareServiceRoles(PerunSession sess) {
 			log.trace("AuthzResolver.init: Perun Registrar {} loaded", perunRegistrars);
 		}
 	}
+
+	/**
+	 * Fetch and update all necessary objects from the given parameters and then resolve access rights for the session's principal.
```

Comment:
This sounds like the method "authorized" is updating something in Perun.

Reply:
Fixed.